flink-mesos/src/main/java/org/apache/flink/mesos/scheduler/Offer.java

```diff
@@ -156,6 +164,16 @@ public long getOfferedTime() {
 		return attributeMap;
 	}
 
+	@Override
+	public Double getScalarValue(String name) {
+		return aggregatedScalarResourceMap.getOrDefault(name, 0.0);
+	}
+
+	@Override
+	public Map<String, Double> getScalarValues() {
+		return aggregatedScalarResourceMap;
```

Comment:
I notice that we return all scalar resource types (`cpus`, `gpus`, ...) here, but in `LaunchableMesosWorker::getScalarRequests` we return only the generic resource types (`gpus`).   Would you please double-check that this is expected by Fenzo?   I wouldn't want Fenzo to double-count the `cpus` or something.

Reply:
You are correct. It can cause confusion to contain an entry for `cpus` in `Offer::aggregatedScalarResourceMap`.  We need to return only generic resource types (other than `cpus`, `mem`, `network`, and `disk`) as we do in `LaunchableMesosWorker::getScalarRequests`.